src/screens/Cart/cart.css

```diff
@@ -0,0 +1,55 @@
+.total-ticket-container{
+	background-color: var(--bg-color-primary);
+	height: fit-content;
+	width: 45rem;
+	min-width:40rem;
+	position: sticky;
+	top: 10.4rem;
+	color: var(--black-color);
```

Comment:
you can write background color and color together, this way the user shouldn't have to go through all the lines in between.